### ISSUE TITLE
About Training Details

### ISSUE THREAD

**Issue by Arican-Lee** [YC-Liang/DRT#9]
Hello author, when I was reproducing your paper Rain100L, the training result did not reach the 37.61 level described in your paper. Can you provide more training details?
For example, why error_ Plot_ Is it necessary for freq=20 and epoch to be 4600? Why achieve New Minimum Error Recorded! Afterwards, there will be no further lag and training_ Image_ Size=56, may this field of view be too small? Can it be enlarged? Or can you provide your trained model?
![d252b7eada5591b7737912dea526cae](https://github.com/YC-Liang/DRT/assets/98862990/24e0fb75-88a6-4c31-b9d6-ce944be046dc)

**Comment by PatrickStar228** [comment]
> Hello author, when I was reproducing your paper Rain100L, the training result did not reach the 37.61 level described in your paper. Can you provide more training details? For example, why error_ Plot_ Is it necessary for freq=20 and epoch to be 4600? Why achieve New Minimum Error Recorded! Afterwards, there will be no further lag and training_ Image_ Size=56, may this field of view be too small? Can it be enlarged? Or can you provide your trained model? ![d252b7eada5591b7737912dea526cae](https://github.com/YC-Liang/DRT/assets/98862990/24e0fb75-88a6-4c31-b9d6-ce944be046dc)


I also encountered this issue, where the results on the three datasets—Test100, Rain100L, and Rain100H—did not meet the values reported in the paper. I adjusted parameters such as batch_size, training_image_size, epochs, and learning rate decay strategies, which allowed Rain100L to achieve the expected PSNR and SSIM. However, I have not yet found the suitable parameters for Test100 and Rain100H.

**Comment by Arican-Lee** [comment]
I have given up reproducing the content of the paper. And now it has switched to researching artificial intelligence GNSS positioning enhancement.



---Original---
From: ***@***.***&gt;
Date: Mon, Dec 22, 2025 21:26 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [YC-Liang/DRT] About Training Details (Issue #9)


PatrickStar228 left a comment (YC-Liang/DRT#9)
  
Hello author, when I was reproducing your paper Rain100L, the training result did not reach the 37.61 level described in your paper. Can you provide more training details? For example, why error_ Plot_ Is it necessary for freq=20 and epoch to be 4600? Why achieve New Minimum Error Recorded! Afterwards, there will be no further lag and training_ Image_ Size=56, may this field of view be too small? Can it be enlarged? Or can you provide your trained model? d252b7eada5591b7737912dea526cae.png (view on web)
  
I also encountered this issue, where the results on the three datasets—Test100, Rain100L, and Rain100H—did not meet the values reported in the paper. I adjusted parameters such as batch_size, training_image_size, epochs, and learning rate decay strategies, which allowed Rain100L to achieve the expected PSNR and SSIM. However, I have not yet found the suitable parameters for Test100 and Rain100H.
